rpmbuild/copr-rpmbuild.spec

```diff
@@ -91,7 +91,7 @@ Summary: copr-rpmbuild with all weak dependencies
 Requires: %{name} = %{version}-%{release}
 Requires: dist-git-client
 
-%if 0%{?fedora}
+%if 0%{?fedora} && 0%{?fedora} < 41
```

Comment:
I tried to remove those packages on my F40, expecting to see `copr-rpmbuild` fail but it didn't. Is it possible we don't need them at all?

Reply:
/bin/yum is needed for Mock bootstrap where bootstrap image is not used; in such case, Mock prefers using yum on host for installing the bootstrap chroot having a correct Yum variant.  Then, installing the buildroot may be done with the right Yum.

Reply:
And as Mirek mentioned, if bootstrap is off entirely.